domutil.go

```diff
@@ -1,12 +1,24 @@
 package vecty
 
+func parentNode(node jsObject) jsObject {
+	if node == nil {
+		panic("vecty: Cannot find parent of nil node")
+	}
+	parent := node.Get("parentNode")
+	if parent == nil {
+		panic("vecty: Missing parent node")
+	}
```

Comment:
Why do we need this function at all (instead of just invoking `node.Get("parentNode)` directly and assuming the result is non-nil. It'll panic just the same, and I hope that real user applications won't ever hit either of these cases.